ElementX/Sources/Services/Timeline/TimelineItemContent/AggregratedReaction.swift

```diff
@@ -23,7 +23,15 @@ struct AggregatedReaction: Hashable {
     /// The reaction that was sent.
     let key: String
     /// The user ids of those who sent the reactions
-    let senders: [String]
+    let senders: [ReactionSender]
+}
+
+/// Details of who sent the reaction
+struct ReactionSender: Hashable {
+    /// The id of the user who sent the reaction
+    let senderId: String
```

Comment:
We generally use `ID` on the Swift side :)
```suggestion
    let senderID: String
```